.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Log file
 *.log
+!important.log
```

Comment:
Do you have this file anywhere?

Reply:
No, I don't.

Reply:
Should I remove this line?

Reply:
Yes, or change it to more realistic file (like report.log)